js/tooltip.js

```diff
@@ -385,13 +385,54 @@
     return delta
   }
 
+  Tooltip.prototype.didModuleReceiveTypeStringOrFunction = function (module, received) {
+    if (!/string|function/.test(received)) {
+      throw new Error(module + ' received type ' + received + '. Was expecting type string or a function.')
+    }
+
+    return true
+  }
+
+  Tooltip.prototype.didFunctionReturnStringOrJQuery = function (module, returned, returnedType) {
```

Comment:
I'm still not happy with these method names. They aren't used anywhere else so why not just inline them into `checkValidTypes`?